content/en/docs/how-tos/interact-with-running-jobs.md

```diff
@@ -0,0 +1,146 @@
+
+```bash
+$ oc -n ci-op-xxxxxxx rsh <pod-name> -c <container-name>
+```
+
+## Access the external cluster launched with your changes
+If you are debugging a job in which an external cluster was launched (`e2e-aws`, `e2e-*`), you might find it useful to access the cluster.
+After the installer pod has successfully completed (usually this is ~30 min after the job was triggered), you will find the test pod running.
+From the project accessed above, (`Home`->`Projects`) you can grab the `KUBECONFIG` for the external cluster and copy it to your local system,
+to access the cluster against which the extended test suites are running.
+Access the `Terminal` tab of the running test pod (`Workloads` -> `Pods` -> `Terminal`). Find the `kubeconfig` file in the pod's `/tmp` directory.
+Once copied to your local system, you can proceed to run `oc` commands against the cluster by setting `KUBECONFIG` environment variable
+or passing `--kubeconfig` to `oc`. Again, this is intended for WIP PRs only. The cluster will be terminated whenever the job completes.
+It is usually more productive to launch a cluster using `cluster-bot` through `Slack` and manually run `openshift-tests` suites against that, rather
+than through a PR job's cluster. For how to run `openshift-tests` binary and to find more information about the test suites, see
+[How to run the test suites myself outside of CI.](#how-to-run-the-test-suites-myself-outside-of-ci)
+
+## How do I know which tests will run?
+
+It can be quite confusing to find the test command that is running in CI. Jobs are configured in
+[the release repository](https://github.com/openshift/release). To give an example, the `e2e-*` tests run the command
+[openshift-tests run "${TEST_SUITE}" ${TEST_ARGS:-}](https://github.com/openshift/release/blob/8ccb2b1c17387920b3de0180c52052c535603855/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh#L142)
+against external clusters. You can find other test commands in
+[release/ci-operator/config/openshift](https://github.com/openshift/release/tree/master/ci-operator/config).
+For example, the `ci/prow/unit` test command for `openshift-apiserver` is
+[make test-unit JUNITFILE=${ARTIFACT_DIR}/junit_report.xml --warn-undefined-variables](https://github.com/openshift/release/blob/8ccb2b1c17387920b3de0180c52052c535603855/ci-operator/config/openshift/openshift-apiserver/openshift-openshift-apiserver-master.yaml#L60-#L62). The job definitions are
+generated from the step-registry workflow and/or the ci-operator/config files. For example, with the `openshift-apiserver` repository, the job yaml
+definitions are found
+[here](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/openshift-apiserver/openshift-openshift-apiserver-master-presubmits.yaml)
+You might grep for `e2e-aws` or `ci/prow/unit` to view an individual job yaml.
+
+## How to run the test suites myself outside of CI
```

Comment:
nit: could we make sure we're using `Proper Title Case` in titles?

Reply:
Yes, I went back and forth with that - ie  # == title  while ## == heading that might not require Title Case - but I'll make it so

Reply:
I've updated them all to be TitleCase